File: src/main/java/com/mintos/weather/entity/IPRegionEntity.java
package com.mintos.weather.entity;

import org.springframework.data.relational.core.mapping.Table;

import java.util.UUID;

@Table("IP_REGION")
public record IPRegionEntity(
    UUID id,
    String ip,
    String region,
    Long lookedUpAtEpoch
) {
}

File: src/main/kotlin/com/mintos/weather/entity/IPRegionEntity.kt
package com.mintos.weather.entity

import org.springframework.data.relational.core.mapping.Table
import java.util.UUID

@Table("IP_REGION")
data class IPRegionEntity(
    val id: UUID,
    val ip: String,
    val region: String,
    val lookedUpAtEpoch: Long,
)